projects/core/src/lib/time/time-step-selector/time-step-selector.component.ts

```diff
@@ -33,6 +38,15 @@ export class TimeStepSelectorComponent implements OnChanges {
             map(([time, format]) => {
                 if (this.timeSteps) {
                     this.currentTimeIndex = this.timeSteps.findIndex((t) => time.isSame(t));
+
+                    if (this.useCeFormat) {
+                        if (time.start.year() < 0) {
+                            format = format.replace('YYYY', 'YYYY [BCE]');
+                        } else {
+                            format = format.replace('YYYY', 'YYYY [CE]');
```

Comment:
should be AD, if they want BC.

Reply:
[2333641](https://github.com/geo-engine/geoengine-ui/pull/681/commits/2333641aa6d94454f4336074999fd4fb3d06603b)